inference/benchmark_video.py

```diff
@@ -33,7 +34,8 @@
     "bf16": lambda module: module.to(dtype=torch.bfloat16),
     "fp8wo": lambda module: quantize_(module, float8_weight_only()),
     "fp8dq": lambda module: quantize_(module, float8_dynamic_activation_float8_weight()),
-    "fp6": lambda module: quantize_(module, fp6_llm_weight_only()),
+    "fp8dqrow": lambda module: quantize_(module, float8_dynamic_activation_float8_weight(granularity=PerRow())),
+    "fp6": lambda module: quantize_(module, fpx_weight_only(3, 2)),
```

Comment:
Let's do this in a separate PR for both `benchmark_video.py` and `benchmark_image.py`?